### ISSUE TITLE
conflict with cmder.exe 

### ISSUE THREAD

**Issue by wqwin** [snowie2000/mactype#377]
The MacType's hooks are detected at 0x00007FFF83450000
         D:\MacType\MacType64.dll
         Please add ConEmuC.exe and ConEmuC64.exe
         to the exclusion list to avoid crashes!
         https://conemu.github.io/en/Installation.html#mactype

**Comment by nguyenbathanh** [comment]
This doesn't affect to your application, just ignore it.

**Comment by DRSDavidSoft** [comment]
Personally I'd still suggest adding these two processes to the exclusion list to remove the unnecessary warning message (as they "do not draw any text on graphical canvas at all", anyway):

```
[UnloadDll]
ConEmuC.exe
ConEmuC64.exe
```

https://github.com/snowie2000/mactype/issues/523#issuecomment-565701316

**Comment by snowie2000** [comment]
It’s not true. Cmder does draw text on its ui and it does get affected by mactype

**Comment by DRSDavidSoft** [comment]
@snowie2000 You're right, it does -- however, those two processes are console-only internal programs that are unrelated to the GUI drawings (`ConEmu.exe` and `ConEmu64.exe` as opposed to `ConEmuC.exe` and `ConEmuC64.exe`)

<sup>**EDIT** just to be clear, [I don't believe that MacType would cause a crash in ConEmu anymore](https://github.com/Maximus5/ConEmu/issues/463#issuecomment-466850376), however, excluding those processes will prevent warning messages from appearing, especially since the console-only programs don't draw any GUI texts.</sup>

I think @Maximus5 can confirm

**Comment by snowie2000** [comment]
Make sense. Accepted. Thank you.
